tests/templates/kuttl/kerberos/02-create-kerberos-secretclass.yaml.j2

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      kubectl apply -n $NAMESPACE -f - <<EOF
+      ---
+      apiVersion: secrets.stackable.tech/v1alpha1
+      kind: SecretClass
+      metadata:
+        name: kerberos-$NAMESPACE
+      spec:
+        backend:
+          kerberosKeytab:
+{% if test_scenario['values']['kerberos-backend'] == 'mit' %}
+            realmName: {{ test_scenario['values']['kerberos-realm'] }}
+            kdc: krb5-kdc.$NAMESPACE.svc.cluster.local
+{% elif test_scenario['values']['kerberos-backend'] == 'activeDirectory' %}
+            realmName: SBLE.TEST
+            # Must be the FQDN of the AD domain controller
+            kdc: sble-adds1.sble.test
+{% endif %}
+            admin:
+              {{ test_scenario['values']['kerberos-backend'] }}:
+{% if test_scenario['values']['kerberos-backend'] == 'mit' %}
+                kadminServer: krb5-kdc.$NAMESPACE.svc.cluster.local
+{% elif test_scenario['values']['kerberos-backend'] == 'activeDirectory' %}
+                # Must be the FQDN of the AD domain controller
+                ldapServer: sble-adds1.sble.test
+                passwordCacheSecret:
+                  name: secret-operator-ad-passwords
+                  namespace: $NAMESPACE
+                # Container must be created manually
+                # Users must be deleted by user between each test run
+                userDistinguishedName: CN=Stackable,CN=Users,DC=sble,DC=test
+                schemaDistinguishedName: CN=Schema,CN=Configuration,DC=sble,DC=test
+{% endif %}
+            adminKeytabSecret:
+              namespace: $NAMESPACE
+              name: secret-operator-keytab
+            adminPrincipal: stackable-secret-operator
+      EOF
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-operator-keytab
+data:
+{% if test_scenario['values']['kerberos-backend'] == 'mit' %}
+  # To create keytab. When promted enter password asdf
+  # cat | ktutil << 'EOF'
+  # list
+  # add_entry -password -p stackable-secret-operator@CLUSTER.LOCAL -k 1 -e aes256-cts-hmac-sha384-192
+  # wkt /tmp/keytab
+  # EOF
+{% if test_scenario['values']['kerberos-realm'] == 'CLUSTER.LOCAL' %}
+  keytab: BQIAAABdAAEADUNMVVNURVIuTE9DQUwAGXN0YWNrYWJsZS1zZWNyZXQtb3BlcmF0b3IAAAABZAYWIgEAFAAgm8MCZ8B//XF1tH92GciD6/usWUNAmBTZnZQxLua2TkgAAAAB
```

Comment:
Probably want to decide how to handle these before merging

Reply:
Do you see any room for improvement? I personally would be fine with merging as-is, but also happy about improvement ideas :see_no_evil:

Reply:
I think the ideal would be to generate the keytabs on demand, but that also brings downsides.. bleh

Reply:
Yeah, it's tricky...